CleverPush/CleverPush/Source/CleverPush.m

```diff
@@ -893,6 +893,25 @@ + (void)subscribe:(CPHandleSubscribedBlock)subscribedBlock {
     }
 }
 
+#pragma mark - update the userdefault value for key @"UNSUBSCRIBED" with the dynamic argument named status.
++ (void)setUnsubscribeStatus:(BOOL)status {
+    [[NSUserDefaults standardUserDefaults] setBool:status forKey:@"UNSUBSCRIBED"];
+    [[NSUserDefaults standardUserDefaults] synchronize];
+}
+
+#pragma mark - get the userdefault value for key @"UNSUBSCRIBED" and prevent subscribe based on the boolean flag.
++ (BOOL)getUnsubscribeStatus {
+    if([[NSUserDefaults standardUserDefaults] objectForKey:@"UNSUBSCRIBED"] != nil) {
+        if([[NSUserDefaults standardUserDefaults] boolForKey:@"UNSUBSCRIBED"]) {
```

Comment:
Can't we directly return `[[NSUserDefaults standardUserDefaults] boolForKey:@"UNSUBSCRIBED"]`?

Also we should prefix the key "UNSUBSCRIBED" like the other keys (CleverPush_UNSUBSCRIBED....)